dates.py

```diff
@@ -0,0 +1,25 @@
+from datetime import date
+from log import logger
+
+
+def day_difference(fd, sd):
+    try:
+        fd_object = datetime.strptime(..)
```

Comment:
It was just an example, here you can read an instruction on how to use it https://timeweb.cloud/tutorials/python/metod-strptime-i-strftime-v-python#metod-strptime